src/components/Workspace.jsx

```diff
@@ -330,8 +366,41 @@ class Workspace extends React.Component {
     );
   }
 
+  _storeColumnRef(index, column) {
+    this.columnRefs[index] = column;
+  }
+
+  _storeDividerRef(divider) {
+    this.dividerRef = divider;
+  }
+
+  _handleDividerStart() {
+    this.props.dispatch(startDragColumnDivider());
+  }
+
+  _handleDividerStop() {
+    this.props.dispatch(stopDragColumnDivider());
+  }
+
+  _handleDividerDrag(_, {deltaX, lastX, x}) {
+    this.props.dispatch(dragColumnDivider({
+      columnWidths: getNodeWidths(this.columnRefs),
+      dividerWidth: getNodeWidth(this.dividerRef),
```

Comment:
I have a mild preference for sticking with the non-plural here; I think it supports the goal of `dragColumnDivider`'s signature clearly defining what situation it's meant for.